src/sass/pages/offering/_value-props.scss

```diff
@@ -68,6 +68,7 @@
         text-align: left;
         font-size: 16px;
         font-weight: 300;
+        padding-top: 8px;
```

Comment:
I'm not sure why, but in Google Chrome I see that for some reason there's already a 6px gap between the image and the caption.
I'm not sure whether the desired 8px in the ticket is accounting for this (and so therefore the 14px we have in this PR is acceptable), or whether we need to compensate for that and just add 2 px?

Reply:
Fixed this to be 2px